inventory/sample.osp.example.com.d/inventory/group_vars/all.yml

```diff
@@ -90,3 +90,17 @@ rhsm_password: ''
 #rhsm_password: ''
 # leave commented out if you want to `--auto-attach` a pool
 #rhsm_pool: ''
+
+# NOTE: these need to be declared in this file because openshift_hosted_registry_storage_kind
+# is needed prior to the creation of OSEv3 hosts. The remainder are here for grouping purposes
+openshift_hosted_registry_storage_kind: openstack
+openshift_hosted_registry_storage_access_modes: ['ReadWriteOnce']
+openshift_hosted_registry_storage_openstack_filesystem: ext4
+openshift_hosted_registry_storage_volume_size: 50Gi
```

Comment:
The `volume_size` should be set based on the `osp_volumes` values (and hence removed from here)